tools/install.sh

```diff
@@ -125,12 +154,57 @@ touch /var/log/proactive/scheduler
 chown -R $USER:$USER /var/log/proactive
 
 if confirm "Start the proactive-scheduler service at startup? [Y/n] " ; then
-    if chkconfig proactive-scheduler on 2> /dev/null ; then
-        echo "proactive-scheduler service added as a startup daemon"
+    if [[ "$OS" == "RedHat" ]]; then
+       chkconfig proactive-scheduler on
+    elif [[ "$OS" == "Debian" ]]; then
+       update-rc.d proactive-scheduler defaults
     else
-        update-rc.d proactive-scheduler defaults
+        echo "Installation as a startup script not supported on this operating system."
     fi
 fi
 
+if which git > /dev/null 2>&1; then
+    # Porting of older version configuration and data files
+
+    cd $PA_ROOT/default/config && git init
+    cd $PA_ROOT/default/config && git config user.email "support@activeeon.com"
```

Comment:
You could move to `$PA_ROOT/default/config` only once. The next repetitions look useless.